projects/js-packages/components/components/toggle-control/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { ToggleControl as CalypsoToggleControl } from '@wordpress/components';
```

Comment:
```suggestion
import { ToggleControl as WPToggleControl } from '@wordpress/components';
```

It isn't Calypso but WP.org